src/world/mod.rs

```diff
@@ -85,6 +85,7 @@ pub trait Builder {
 ///     .with(Pos { x: 1.0, y: 3.0 })
 ///     .build(); // Returns the `Entity`
 /// ```
+#[must_use]
```

Comment:
I think there was a way to provide an explanation for this. Would you mind adding that?